### ISSUE TITLE
False positive for experimental:indent with new line before return type

### ISSUE THREAD

**Issue by lwasyl** [pinterest/ktlint#649]
For the given function:
```kotlin
abstract fun doPerformSomeOperation(param: ALongParameter):
    SomeLongInterface<ALongParameter.InnerClass, SomeOtherClass>
```
and with both indent and continuation indent set to `4`, KtLint reports at the second line that `Unexpected indentation (8) (should be 4)`

This seems to be a false positive, since _some_ indent (normal or continuation one) should be applied for that line, otherwise we'd end up with
```kotlin
abstract fun doPerformSomeOperation(param: ALongParameter):
SomeLongInterface<ALongParameter.InnerClass, SomeOtherClass>
```
